test/basic_app/splat.yaml

```diff
@@ -13,21 +13,26 @@ options:
   undefined_syms_auto_path:  split/undefined_syms_auto.txt
   asset_path: split/assets
   asm_generated_by: False
+  compiler: GCC
 segments:
-  - name: basic_app
+  - name: header
+    type: header
+    start: 0x00
+  - name: dummy_ipl3
     type: code
-    start: 0x00000000
-    vram:  0x00400000
-    bss_size: 0x24
-    subalign: 4
+    start: 0x40
+    vram: 0xA4000040
     subsegments:
-      - [0x0, data, .MIPS.abiflags]
-      - [0x1dc, hasm, mymb]
-      - [0x204, c, main]
-      - [0x230, data]
-      - [0x12D4, bin, bin_data]
-      - [0x12D8] # intentionally short since 0x12DC seems to break spimdisasm bss output
-      - { type: bss, vram: 0x004012dc, name: __bss_start }
-      - { type: bss, vram: 0x004012e0, name: dtor_idx.3753 }
-      - { type: bss, vram: 0x004012e4, name: object.3763 }
-  - [0x12DC]
+      - [0x0040, bin, dummy_ipl3]
+  - name: boot
+    type: code
+    start: 0x1000
+    vram:  0x80000400
+    bss_size: 0x80
+    subsegments:
+      - [0x1000, c, main]
+      - [0x10F0, hasm, handwritten]
+      - [0x1100, data, main]
+      - [0x1110, rodata, main]
```

Comment:
The expected folder seems to be missing the asm/data/main.rodata.s file that this generates.

Reply:
Shouldn't the CI fail if the expected folder does not have the generated files by this yaml? Or did I misunderstood something? :thinking:

Reply:
That could be a splat bug. The file gets generated if you add the line `      - [0x1138]` after the rodata line, but that shouldn't be needed from my understanding. It might be best to just add that to the test yaml for now with a TODO to remove it when that's resolved in splat.

Reply:
I added the line and committed the differences. It causes a number of things to change. Does this all look right? https://github.com/ethteck/splat/pull/244/commits/3a9bf1c619eb7961012612990cab753d24f98edd

Reply:
Yeah, those are more correct now as the jump table is actually getting migrated.